qemu/tests/cfg/qsd_block_commit.cfg

```diff
@@ -0,0 +1,51 @@
+- qsd_block_commit:
+    type = qsd_block_commit
+    virt_test_type = qemu
+    only Linux
+    images += " stg1"
+    boot_drive_stg1 = no
+    start_vm = no
+    kill_vm = yes
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    image_size_stg1 = 2G
+    image_name_stg1 = stg1
+    snapshot_tags = sn1 sn2 sn3 sn4
+
+    image_name_sn1 = sn1
+    image_format_sn1 = qcow2
+
+    image_name_sn2 = sn2
+    image_format_sn2 = qcow2
+
+    image_name_sn3 = sn3
+    image_format_sn3 = qcow2
+
+    image_name_sn4 = sn4
+    image_format_sn4 = qcow2
+
+    device_tag = "stg1"
+    format = qcow2
+    rebase_mode = unsafe
+    !Host_RHEL.m7:
+        node = ${device}
+        qemu_force_use_drive_expression = no
```

Comment:
Do we still need this?
This case is for qsd testing, it seems not to run qsd on rhel7, so just set node here

Reply:
We don't run qsd on RHEL7 at present, so remove it as your suggestion.